### PR TITLE
Graph fix

### PR DESCRIPTION
**Pull request by akinolu52**
## fix issue on graph

### CODE REVIEW

File: re_data_ui/src/partials/Graph.tsx [hunk 1]
```diff
@@ -30,12 +30,15 @@ type GenerateGraphResponseProps = {
 }
 
 const getAlertData = (modelId: string, aggregatedModels: Map<string, ReDataModelDetails>) => {
-  const {
-    anomalies,
-    schemaChanges,
-  } = aggregatedModels.get(modelId) as ReDataModelDetails;
+  if (aggregatedModels.has(modelId)) {
```
Comment (mateuszklimek) [inline re_data_ui/src/partials/Graph.tsx:33]: @akinolu52 one question regarding that, I believe this fix is no longer needed after changed to mapping?
Maybe we should just revert to the previous version here / which just fails?

File: re_data_ui/src/partials/Graph.tsx [hunk 2]
```diff
@@ -83,94 +87,89 @@ const generateGraph = (
...
-          schemaChanges: parentSchemaChanges?.length > 0,
-        });
-
-        nodes.push({
-          label: parentModelId,
-          value: parentModelId,
-        });
-        elements.push(parentNode);
-        elementObj[parentModelId] = key?.toString();
+    if (!nodeSet.has(modelId)) {
```
Comment (mateuszklimek) [inline re_data_ui/src/partials/Graph.tsx:90]: similarly here?

File: re_data_ui/src/partials/Graph.tsx (hunk 1 above)
Comment (akinolu52) [reply re_data_ui/src/partials/Graph.tsx:33]: or maybe we should leave it as it's an extra check & it does no harm @mateuszklimek

Comment (mateuszklimek) [reply re_data_ui/src/partials/Graph.tsx:33]: ok, let's do that ;)

**Pull request merged**